more/free-programming-cheatsheets.md

```diff
@@ -128,6 +128,8 @@
 * [Big O Notation Cheat Sheet](https://algodaily.com/lessons/big-o-notation-cheat-sheet)
 * [Data Structures and Algorithms Cheat Sheet - Cheatography](https://cheatography.com/burcuco/cheat-sheets/data-structures-and-algorithms/)
 * [Data structures and algorithms study cheatsheets for coding interviews](https://www.techinterviewhandbook.org/algorithms/study-cheatsheet/)
+* [Striver 79](https://takeuforward.org/interview-sheets/strivers-79-last-moment-dsa-sheet-ace-interviews)
+* [Striver SDE Sheet](https://takeuforward.org/interviews/strivers-sde-sheet-top-coding-interview-problems/)
```

Comment:
```suggestion
* [79 Last Moment DSA Sheet – Ace Interviews](https://takeuforward.org/interview-sheets/strivers-79-last-moment-dsa-sheet-ace-interviews) - Striver
* [Top Coding Interview Problems](https://takeuforward.org/interviews/strivers-sde-sheet-top-coding-interview-problems/) - Striver
```
I think formatting something like this makes more sense. Striver is actually the author so the other part of the title seems more important.

Reply:
Do as suggested!